app/views/layouts/doorkeeper.html.haml

```diff
@@ -15,14 +15,15 @@
     = javascript_pack_tag 'doorkeeper', 'data-turbolinks-track': 'reload'
 
     = csrf_meta_tags
-  %body.h-full.flex.flex-col.bg-light
+  %body.h-full.flex.flex-col.bg-gray-100.dark:bg-gray-900
     .flex.flex-col.flex-grow.justify-center
-      %img.w-64.mx-auto.p-4{src:"https://public.svsticky.nl/logos/logo_compact_kleur.svg"}
+      %img.w-64.mx-auto.p-4.dark:hidden{src:"https://public.svsticky.nl/logos/logo_compact_kleur.svg"}
+      %img.w-64.mx-auto.p-4.hidden.dark:block{src:"https://public.svsticky.nl/logos/logo_compact_outline_wit.svg"}
 
-      .shadow.px-10.py-4.rounded-md.bg-white.mx-auto
+      .shadow.px-10.py-4.rounded-md.bg-white.mx-auto.dark:bg-gray-800.dark:shadow-
```

Comment:
Shouldn't this be dark:shadow? I'm not experienced with tailwind but the syntax looks weird to me